.github/workflows/src/summarize-checks/summarize-checks.js

```diff
@@ -35,6 +41,11 @@ import {
   typeSpecRequirementDataPlaneTsg,
 } from "./tsgs.js";
 
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { execSync } from 'child_process';
```

Comment:
Use this helper instead.  It uses the most secure exec API:

https://github.com/Azure/azure-rest-api-specs/blob/2ae7cdd6509fb91ed9d7742186e278b67b86398e/.github/shared/src/exec.js#L37-L46